doc/content/getting-started/programming-software/st-examples/stcubeide/_index.md

```diff
@@ -7,10 +7,26 @@ distributions: null
 
 If you are using [STM32CubeIDE](https://www.st.com/en/development-tools/stm32cubeide.html) as your build environment, then using the same tool to program your {{% gnse %}} is convenient.
 
-<!--more-->
```

Comment:
Why remove this line?

This line protects the documentation framework from having all the page information on the thumbnail.

See how this looks now 

![Screen Shot 2021-09-13 at 18 01 19](https://user-images.githubusercontent.com/24204263/133119851-f42cdc98-98f2-459f-80d5-abd0507c51d8.png)

in comparison to the old [overview](https://www.genericnode.com/docs/getting-started/programming-software/st-examples/)

Reply:
fixed